src/routes/(admin)/questionbank/+page.svelte

```
@@ -1,35 +1,107 @@
+	// Check if filters are applied
+	const hasActiveFilters = $derived(() => {
+		const meaningfulParams = ['search', 'name', 'tag_ids', 'state_ids'];
+		return (
+			meaningfulParams.some((param) => page.url.searchParams.has(param)) ||
+			filteredTags.length > 0 ||
+			filteredStates.length > 0
+		);
+	});
+
+	// Render expanded row content
+	function expandedRowContent(question: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)